src/app/frontend/logs/logs.html

```diff
@@ -55,13 +55,21 @@
                             'kd-logs-color-icon': !ctrl.logsService.getInverted()}">
           format_color_text
         </md-icon>
+        <md-tooltip md-delay="500"
+                    md-autohide>
+          [[Invert colors|Tooltip for button that toggles between light & dark background in log viewer.]]
```

Comment:
Run `gulp serve:prod` or `gulp generate-xtbs` to generate translations and commit them.

Reply:
Probably done? Not sure what the expected output is, but there are some new changes.

Reply:
It generates translation entries :) Thanks, I will check tomorrow.